### ISSUE TITLE
Problems with sorting

### ISSUE THREAD

**Issue by Milincho** [Radarr/Radarr#8662]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Sorting by year shows redundant info:
![image](https://github.com/Radarr/Radarr/assets/60118859/b32c8478-fcfe-479c-8a18-eb28d5022998)
The third line with the year of the movie is not available as an enable/disable option and that year info is just redundant with the "Show Cinema release date" option, which already shows the year.

Sorting by "In Cinemas" puts movies never released in cinemas first:
![image](https://github.com/Radarr/Radarr/assets/60118859/639d66a0-f6a6-4089-ac01-3715592ace29)

Unreleased movies with a future set cinema release date should take precedence over digital releases with a previous date...
With the current sorting method any direct-to-dvd movie from 20 years ago gets presented before any current or future cinema releases...

### Expected Behavior

Unreleased movies with a future set cinema release date should take precedence over digital releases with a previous date...
With the current sorting method any direct-to-dvd movie from 20 years ago gets presented before any current or future cinema releases...

### Environment

```markdown
- OS: Windows 11
- Radarr: 4.6.0.7439
- Docker Install: 
- Using Reverse Proxy: 
- Browser: 
- Database:
```

### What branch are you running?

Develop

### Trace Logs?

![image](https://github.com/Radarr/Radarr/assets/60118859/a68da95b-1e84-49a6-88dc-4dac4b4f973d)
![image](https://github.com/Radarr/Radarr/assets/60118859/15d5caf5-acd5-4ecf-a906-01c6e4d1cfea)

### Trace Logs have been provided as applicable. Reports may be closed if the required logs are not provided.

- [X] I have followed the steps in the wiki link above and provided the required trace logs that are relevant and show this issue.

**Comment by bakerboy448** [comment]
- the year shows as you're sorting by a year/date

**Comment by Milincho** [comment]
Could be an option like the others?

**Comment by Starstrike** [comment]
Unfortunately it appears this change has broken sorting by Digital Release date. I get what OP was trying to accomplish, however, when I sort by Digital Release, I'm seeing that something releasing in September is now sooner than something releasing this week.

Personally, I sort by Digital Release primarily with those dates showing at the top, sorted by what's coming out first. Now, that same method puts the Digital Release dates at the bottom and anything without a Digital Release date sorted before it.

Screenshots are attached that show the improper behaviors. When sorting as shown in the first image, there's a huge gap of no release dates before it shows ones that do have one (sorted properly with Thursday coming before Sep 3 2023). When sorting as shown in the second image, Thursday should come before Sep 3 2023 while leaving everything without a release date below it.

![image](https://github.com/Radarr/Radarr/assets/1996734/c3769f97-cadc-4554-ab7f-d0d66cdf0f91)
![image](https://github.com/Radarr/Radarr/assets/1996734/c3581353-3fff-4de1-9631-8bd51a174e38)

**Comment by DrStrange** [comment]
I respectfully do not agree with the OP that there was a problem with sorting. 
When I sort by digital release date I expect the movies to be brought to the top in release date soonest to last order. Now with 4.6.1.7456, these movies are at the bottom. Sorting in reverse digital release does bring the movies to the top but they are in last to soonest order which is not my preferred order neither. 
There is now a similar problem when sorting by cinema release. Using these sort order I expect the movies to be displayed with those with a cinema release date first, from earliest to latest, then those with no cinema release date displayed next. Now with 4.6.1.7456, this sort order first displays movies with no cinema release date first, then movies that do have a cinema release date are displayed at the bottom. 

Inconvenient if you have a few hundred movies in your database that you have to scroll past to see the digital/cinema release date.

**Comment by Milincho** [comment]
> I respectfully do not agree with the OP that there was a problem with sorting. When I sort by digital release date the expected behaviour is that the movies should be displayed first in release date soonest to last order, then those movies with no digital release date to be displayed next. Now with 4.6.1.7456, movies with a digital release date are displayed after those movies that have no digital release date. Sorting in reverse digital release does bring the movies with a digital release date to the top but they are in last to soonest order whereas I want them displayed from soonest to last. There is also now a similar problem when sorting by cinema release. Using this sort order the expected behaviour is that this sort order should display movies with a cinema release date first, from earliest to latest, then those with no cinema release date displayed next. Now with 4.6.1.7456, this sort order displays movies with no cinema release date first, then movies that do have a cinema release date are displayed next.
> 
> Inconvenient if you have >200 movies in your database that you have to scroll past when you need to see the digital/cinema release date.

That is a different issue/sorting preference that could have its own option.
There really was a sorting problem when titles with no dates were put first, like sorting by "In Cinemas" putting movies that don't have a date (because they were never released in cinemas) in front of the ones that have a date.

**Comment by DrStrange** [comment]
@Milincho, Yes I agree with you. Tetris seems to be released only at a festival, not in cinema, but I have around 60 movies with a release date out of 280 total movies so some of the remaining 220 with no cinema release date I expect will have a release date eg Rendezvous With Rama, so I have opened a new issue as I think #8662 has broken the sort order.

**Comment by Starstrike** [comment]
> @Starstrike what version of Radarr are you using?

Sorry I had notifications on but didn't get one. I am currently on 4.6.1.7456 Develop and observed the behavior shown in the screenshot I shared. I see that #8679 does not look to be fixing this so the "new" behavior will be that items without a release date, when viewing by ascended, will be put first? I, and others here, don't really care for this behavior and an option to switch back would be nice. Having to scroll to the bottom of a potentially very long list seems counter-intuitive here. Items without a release date don't matter as those that do.
